enterprise_gateway/services/kernels/remotemanager.py

```diff
@@ -418,8 +434,30 @@ def _get_process_proxy(self):
            instantiated.
         """
         process_proxy_cfg = get_process_proxy_config(self.kernel_spec)
+        print(self.kernel_spec)
         process_proxy_class_name = process_proxy_cfg.get('class_name')
         self.log.debug("Instantiating kernel '{}' with process proxy: {}".
                        format(self.kernel_spec.display_name, process_proxy_class_name))
         process_proxy_class = import_item(process_proxy_class_name)
         self.process_proxy = process_proxy_class(kernel_manager=self, proxy_config=process_proxy_cfg.get('config'))
+
```

Comment:
So looking into the code, seems kernelspec manager is already handled by the base class of RKM, and it gets invoked the same way as the one attached to RMKM